### ISSUE TITLE
aria/aria-description_attribute - link has the wrong label

### ISSUE THREAD

**Issue by benjiallen** [accessibilitysupported/a11ysupport.io#195]
### Steps to reproduce

- Go to [aria-description attribute](https://a11ysupport.io/tech/aria/aria-description_attribute)
- Observe that the link at the bottom of the page, under the References heading, reads "DRAFT ARIA spec for aria-describedby" but goes to information about `aria-description`

### Suggested fix

Link label should be updated to "DRAFT ARIA spec for aria-description"

**Comment by benjiallen** [comment]
Looks like the file is here: data/tech/aria/aria-description_attribute.json

Line 8.

Let me know if you want me to fix this in a PR.

**Comment by mfairchild365** [comment]
Oh! Thank you for finding that and raising the issue! A PR is most appreciated.

**Comment by benjiallen** [comment]
A PR is in.
